index.html

```diff
@@ -18,18 +18,24 @@
 
 </head>
 
-<body>
+<body class="standard">
     <div id = "header">
+        <div class="flexrow-container">
+            <p>Select a theme:</p>
```

Comment:
This line can be removed since it is very intuitive.

Also, as they say: *Minimalism is the key.*

Reply:
I was actually struggling to find a good sentence for that, so no sentence at all seems perfect! XD